src/rest-client-v5.ts

```diff
@@ -1712,6 +1779,32 @@ export class RestClientV5 extends BaseRestClient {
     return this.get('/v5/spot-margin-trade/data', params);
   }
 
+  /**
+   * Get Historical Interest Rate
+   * You can query up to six months borrowing interest rate of Margin trading.
+   * INFO: Need authentication, the api key needs "Spot" permission. Only supports Unified account.
+   */
+  getHistoricalInterestRate(params: {
```

Comment:
I saw you didnt merge last PR yet, so this might show you missing. I just moved it to the right spot. 

Last PR I just added this function, but it was in the wrong spot. I just moved it now.

Reply:
Merged previous pr

Reply:
Make sure you hit the update branch button in the pr when you're done pushing commits